### ISSUE TITLE
Naming inconsistencies in pyme modules

### ISSUE THREAD

**Issue by zacsimile** [python-microscopy/python-microscopy#216]
We're introducing `pymeclusterofone` as yet another module command to launch elements of PYME. This is inconsistent with the other module names, which are all inconsistent with one another. To make entry points easier for new users, I propose the launch names be as follows.

```
PYMEAcquire -> pyme-acquire
VisGUI -> pyme-visualize
dh5view -> pyme-image
pymeclusterofone -> pyme-cluster
launchWorkers -> pyme-404
```

It would be best to change `pyme-visualize` to `pyme-points` since both `pyme-visualize` and  `pyme-image` are for visualization and analysis, just of different data types.

**Comment by David-Baddeley** [comment]
It's a work in progress ... unfortunately hyphens in executable names aren't safe across platforms. 

All new entry points (including all the cluster ones) follow the pattern:
`PYMECommand`

`PYMEClusterOfOne` can't /shouldn't change to `PYMECluster` as we should be distinguishing between it and an actual cluster

`VisGUI` and `dh5view` definitely need to change (that's been on the TODO list for some time). The main reason it hasn't happened already is that I've been toying with the idea of combining them under a single entry point which would guess the data type and load the appropriate viewer. Do you have any feelings as to whether you think there is actually a need to two separate entry points here?

I'm not super interested in changing launchWorkers, fitMon, taskServerZC etc ... as I expect them to disappear.

**Comment by David-Baddeley** [comment]
ps - I like `PYMEImage` for `dh5view`

**Comment by zacsimile** [comment]
I agree that a single entry point makes the most sense.

I expect them to disappear, too. I was trying to make a 404 Not Found joke with that suggestion. I'll try to refrain from making deadpan jokes in issues from now on, but no promises.

**Comment by zacsimile** [comment]
Addressed in #302.
